### PR TITLE
ThreatIntelligence APIs to v1.0

### PR DESCRIPTION
**Pull request by joerattazzi-microsoft**
Preparing for the ThreatIntelligence APIs for v1.0.

### CODE REVIEW

**Comment by dodaromike** [comment]
Changelog to be added.

**Comment by dodaromike** [comment]
If these 22 new list and get items are all navigation properties of the Security namespace, they need to be added to their parent entities.

File: changelog/Microsoft.ThreatIntelligence.json [hunk 1]
```diff
@@ -1,255 +1,520 @@
...
-  
+  "changelog": [
+    {
+      "ChangeList": [
+        {
+          "Id": "01daeb88-0ba8-4bcf-b85d-5130e7813a0d",
+          "ApiChange": "Resource",
+          "ChangedApiName": "entity",
+          "ChangeType": "Addition",
+          "Description": "Added the [entity](https://learn.microsoft.com/en-us/graph/api/resources/entity?view=graph-rest-1.0) resource",
```
Comment (joerattazzi-microsoft) [inline changelog/Microsoft.ThreatIntelligence.json:10]: Graph Studio generated this. I don't know why it would be showing as new

**Review by dodaromike — APPROVED** [review]

File: api-reference/v1.0/resources/security-host.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
+---
+title: "host resource type"
+description: "Represents a hostname or IP address that is currently or was previously available on the Internet and Microsoft Defender Threat Intelligence has detected."
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-host.md:3]:
```suggestion
description: "Represents a hostname or IP address that is currently or was previously available on the internet and Microsoft Defender Threat Intelligence has detected."
```

File: api-reference/v1.0/resources/security-host.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+doc_type: resourcePageType
+---
+
+# host resource type
+
+Namespace: microsoft.graph.security
+
+[!INCLUDE [threatintelligence-api-disclaimer](../../includes/threatintelligence-api-disclaimer.md)]
+
+Represents a [hostname](../resources/security-hostname.md) or [IP address](../resources/security-ipaddress.md) that is currently or was previously available on the Internet and Microsoft Defender Threat Intelligence has detected.
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-host.md:16]:
```suggestion
Represents a [hostname](../resources/security-hostname.md) or [IP address](../resources/security-ipaddress.md) that is currently or was previously available on the internet and Microsoft Defender Threat Intelligence has detected.
```

File: api-reference/v1.0/resources/security-host.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+| [List trackers](../api/security-host-list-trackers.md)                   | [microsoft.graph.security.hostTracker](../resources/security-hosttracker.md) collection           | Get a list of **hostTracker** resources.                                                                          |
+
+## Properties
+
+| Property          | Type           | Description                                                                                                                                                                                                                             |
+| :---------------- | :------------- | :-------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| firstSeenDateTime | DateTimeOffset | The first date and time when this host was observed. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.       |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-host.md:41]:
```suggestion
| firstSeenDateTime | DateTimeOffset | The first date and time this host was observed. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`.       |
```

File: api-reference/v1.0/resources/security-host.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
...
+| id                | String         | Unique identifier for the host. Read-only. Inherited from [microsoft.graph.security.artifact](../resources/security-artifact.md).                                                                                                       |
+| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this host was observed. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-host.md:43]:
```suggestion
| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this host was observed. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`. |
```

File: api-reference/v1.0/resources/security-hostcomponent.md [hunk 6]
```diff
@@ -0,0 +1,68 @@
...
+| firstSeenDateTime | DateTimeOffset | The first date and time when this web component was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.       |
+| id                | String         | A system-generated ID for this **hostComponent**. Inherited from [microsoft.graph.security.artifact](../resources/security-artifact.md).                                                                                                                                                   |
+| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this web component was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hostcomponent.md:35]:
```suggestion
| lastSeenDateTime  | DateTimeOffset | The most recent date and time when Microsoft Defender Threat Intelligence observed this web component. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`. |
```

File: api-reference/v1.0/resources/security-hostcomponent.md [hunk 7]
```diff
@@ -0,0 +1,68 @@
...
+| :---------------- | :------------- | :----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| category          | String         | The type of component that was detected (for example, `Operating System`, `Framework`, `Remote Access`, or `Server`).                                                                                                                                                                      |
+| firstSeenDateTime | DateTimeOffset | The first date and time when this web component was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.       |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hostcomponent.md:33]:
```suggestion
| firstSeenDateTime | DateTimeOffset | The first date and time when Microsoft Defender Threat Intelligence observed this web component. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`.       |
```

File: api-reference/v1.0/resources/security-hostcookie.md [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+| :---------------- | :------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| domain            | String         | The URI for which the cookie is valid.                                                                                                                                                                                                                                                      |
+| firstSeenDateTime | DateTimeOffset | The first date and time when this **hostCookie** was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.       |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hostcookie.md:33]:
```suggestion
| firstSeenDateTime | DateTimeOffset | The first date and time when this **hostCookie** was observed by Microsoft Defender Threat Intelligence. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`.       |
```

File: api-reference/v1.0/resources/security-hostcookie.md [hunk 9]
```diff
@@ -0,0 +1,66 @@
...
+| id                | String         | A system-generated ID for this **hostCookie**. Inherited from [microsoft.graph.security.artifact](../resources/security-artifact.md).                                                                                                                                                       |
+| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this **hostCookie** was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hostcookie.md:35]:
```suggestion
| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this **hostCookie** was observed by Microsoft Defender Threat Intelligence. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`. |
```

File: api-reference/v1.0/resources/security-hostname.md [hunk 10]
```diff
@@ -0,0 +1,76 @@
...
+
+## Properties
+
+| Property          | Type           | Description                                                                                                                                                                                                                                                                                                                                             |
+| :---------------- | :------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------ |
+| firstSeenDateTime | DateTimeOffset | The first date and time that this [host](../resources/security-host.md) was observed. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. Inherited from [microsoft.graph.security.host](../resources/security-host.md).       |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hostname.md:37]:
```suggestion
| firstSeenDateTime | DateTimeOffset | The first date and time that this [host](../resources/security-host.md) was observed. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. Inherited from [microsoft.graph.security.host](../resources/security-host.md).       |
```

File: api-reference/v1.0/resources/security-hostname.md [hunk 11]
```diff
@@ -0,0 +1,76 @@
...
+| id                | String         | Unique identifier for the **hostname**. Read-only. Inherited from [microsoft.graph.security.artifact](../resources/security-artifact.md).                                                                                                                                                                                                               |
+| lastSeenDateTime  | DateTimeOffset | The most recent date and time that this [host](../resources/security-host.md) was observed. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. Inherited from [microsoft.graph.security.host](../resources/security-host.md). |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hostname.md:39]:
```suggestion
| lastSeenDateTime  | DateTimeOffset | The most recent date and time that this [host](../resources/security-host.md) was observed. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`. Inherited from [microsoft.graph.security.host](../resources/security-host.md). |
```

File: api-reference/v1.0/resources/security-hosttracker.md [hunk 12]
```diff
@@ -0,0 +1,66 @@
...
+
+## Properties
+
+| Property          | Type           | Description                                                                                                                                                                                                                                                                                  |
+| :---------------- | :------------- | :------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| firstSeenDateTime | DateTimeOffset | The first date and time when this **hostTracker** was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.       |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hosttracker.md:32]:
```suggestion
| firstSeenDateTime | DateTimeOffset | The first date and time when this **hostTracker** was observed by Microsoft Defender Threat Intelligence. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`.       |
```

File: api-reference/v1.0/resources/security-hosttracker.md [hunk 13]
```diff
@@ -0,0 +1,66 @@
...
+| id                | String         | A system-generated ID for this **hostTracker**. Inherited from [microsoft.graph.security.artifact](../resources/security-artifact.md).                                                                                                                                                       |
+| kind              | String         | The kind of **hostTracker** that was detected. For example, `GoogleAnalyticsID` or `JarmHash`.                                                                                                                                                                                               |
+| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this **hostTracker** was observed by Microsoft Defender Threat Intelligence. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`. |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-hosttracker.md:35]:
```suggestion
| lastSeenDateTime  | DateTimeOffset | The most recent date and time when this **hostTracker** was observed by Microsoft Defender Threat Intelligence. The timestamp type represents date and time information using ISO 8601 format and is always in UTC. For example, midnight UTC on Jan 1, 2014, is `2014-01-01T00:00:00Z`. |
```

File: api-reference/v1.0/resources/security-intelligenceprofile.md [hunk 14]
```diff
@@ -0,0 +1,83 @@
...
+
+Namespace: microsoft.graph.security
+
+[!INCLUDE [threatintelligence-api-disclaimer](../../includes/threatintelligence-api-disclaimer.md)]
+
+The Microsoft Defender Threat Intelligence Profiles (Intel Profile) API provides the most up-to-date threat actor infrastructure visibility in the industry today, enabling threat intelligence and security operations (SecOps) teams to streamline their advanced threat hunting and analysis workflows. These teams have historically struggled to obtain visibility into the full extent of nation state and cybercriminal adversary infrastructures, creating blind spots in detection and response automation, hunting, and analytics. Sophisticated security professionals can use Microsoft Intelligence Profile APIs, along with the [indicators](../resources/security-intelligenceprofileindicator.md) and other associated markers, to automate defense of their organizations and track potentially malicious activity targeting their organization or industry. Users of the Microsoft Defender Threat Intelligence Profiles APIs have access to detailed threat actor intel profiles, including background information and interpretation guidance.
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-intelligenceprofile.md:16]:
```suggestion
The Microsoft Defender Threat Intelligence Profiles (Intel Profile) API provides the most up-to-date threat actor infrastructure visibility in the industry today, enabling threat intelligence and security operations (SecOps) teams to streamline their advanced threat hunting and analysis workflows. These teams have historically struggled to obtain visibility into the full extent of nation state and cybercriminal adversary infrastructures, creating blind spots in detection and response automation, hunting, and analytics. Sophisticated security professionals can use Microsoft Intelligence Profile APIs, along with the [indicators](../resources/security-intelligenceprofileindicator.md) and other associated markers, to automate defense of their organizations and track potentially malicious activity targeting their organization or industry. Users of the Microsoft Defender Threat Intelligence Profiles APIs have access to detailed threat actor intel profiles, including background information and interpretation guidance.
```
```suggestion
The Microsoft Defender Threat Intelligence Profiles (Intel Profiles) API provides the most up-to-date threat actor infrastructure visibility in the industry today, enabling threat intelligence and security operations (SecOps) teams to streamline their advanced threat hunting and analysis workflows. These teams have historically struggled to obtain visibility into the full extent of nation state and cybercriminal adversary infrastructures, creating blind spots in detection and response automation, hunting, and analytics. Sophisticated security professionals can use Microsoft Intelligence Profile APIs, along with the [indicators](../resources/security-intelligenceprofileindicator.md) and other associated markers, to automate defense of their organizations and track potentially malicious activity targeting their organization or industry. Users of the Microsoft Defender Threat Intelligence Profiles APIs have access to detailed threat actor intel profiles, including background information and interpretation guidance.
```

File: api-reference/v1.0/resources/security-intelligenceprofile.md [hunk 15]
```diff
@@ -0,0 +1,83 @@
...
+| description                | [microsoft.graph.security.formattedContent](../resources/security-formattedcontent.md)                                                                | A synopsis of the threat actor. This property places the threat actor in wider context, tracing its discovery, history, significant campaigns, targeting, techniques of note, affiliations with governments, law enforcement countermeasures, and any areas of dispute among the security community regarding attribution. |
+| firstActiveDateTime        | DateTimeOffset                                                                                                                                        | The date and time when this **intelligenceProfile** was first active. The Timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.                                                                         |
```
Comment (Danielabom) [inline api-reference/v1.0/resources/security-intelligenceprofile.md:32]:
```suggestion
| firstActiveDateTime        | DateTimeOffset                                                                                                                                        | The date and time when this **intelligenceProfile** was first active. The timestamp type represents date and time information using ISO 8601 format and is always in UTC time. For example, midnight UTC on Jan 1, 2014 is `2014-01-01T00:00:00Z`.                                                                         |
```

File: api-reference/v1.0/resources/security-threatintelligence-overview.md [hunk 16]
```diff
@@ -0,0 +1,42 @@
...
+ms.localizationpriority: medium
+ms.prod: "security"
+doc_type: conceptualPageType
+---
+
+# Use the Microsoft Graph APIs for Microsoft Defender Threat Intelligence
+
+[!INCLUDE [threatintelligence-api-disclaimer](../../includes/threatintelligence-api-disclaimer.md)]
+
+Organizations conducting threat infrastructure analysis and gathering threat intelligence can use [Microsoft Defender Threat Intelligence](/defender/threat-intelligence/what-is-microsoft-defender-threat-intelligence-defender-ti) (Defender TI) to streamline triage, incident response, threat hunting, vulnerability management, and cyber threat intelligence analyst workflows. In addition, you can use the APIs exposed by Microsoft Defender Threat Intelligence on Microsoft Graph to deliver world-class threat intelligence that helps protect your organization from modern cyber threats. You can identify adversaries and their operations, accelerate detection and remediation, and enhance your security investments and workflows.
```
Comment (joerattazzi-microsoft) [inline api-reference/v1.0/resources/security-threatintelligence-overview.md:14]: I know the URL link is failing the API Doctor. This is copy/pasted from the `beta` docs - original change was made by the documentation team, here: https://github.com/microsoftgraph/microsoft-graph-docs/pull/21163

**Comment by joerattazzi-microsoft** [comment]
> Please remove the snippets.

@Danielabom  - done

File: api-reference/v1.0/resources/security-threatintelligence-overview.md (hunk 16 above)
Comment (joerattazzi-microsoft) [reply api-reference/v1.0/resources/security-threatintelligence-overview.md:14]: @Danielabom - this is the last markdown error ApiDoctor is complaining about.

**Review by Lauragra — APPROVED** [review]

**Pull request merged**